### ISSUE TITLE
Tab behaviour is not very intuitive

### ISSUE THREAD

**Issue by dbogdanov** [benji300/joplin-note-tabs#28]
Thanks for developing this excellent plugin!

Often I am in a situation where I have a tab open, but I want to get rid of it entirely and switch to previous tab. There's no easy way to do it. The only option I have is to pin it. 

At the same time, for a pinned tab, clicking on unpin makes it an active unpinned tab while removing the previous active unpinned tab. This is not intuitive.

Would it make sense to allow multiple unpinned tabs? A user will be able to click on any of them and change their content, and the tabs can be easily managed by pin/unpin and close buttons (Is that possible to implement them with mouse right click?)

**Comment by benji300** [comment]
"_Often I am in a situation where I have a tab open, but I want to get rid of it entirely and switch to previous tab._"
- Does the temporary tab really bother you or is it more of an inconvenience? See https://discourse.joplinapp.org/t/plugin-note-tabs/12752/75. 

"_At the same time, for a pinned tab, clicking on unpin makes it an active unpinned tab while removing the previous active unpinned tab. This is not intuitive._"
- Why is it not intuitive? This behavior applies only for the selected note. So after unpinning the selected note, the note still remains selected. So there's a need for a tab for this note. Unselected/inactive tabs will be removed completely when unpinned. 
Anyway, have you read this [post](https://discourse.joplinapp.org/t/plugin-note-tabs/12752/81?u=benji300) on the forum? I have created a feature [#25](https://github.com/benji300/joplin-note-tabs/issues/25) to change this behavior in a configurable way. Would this fit your needs?

"_Would it make sense to allow multiple unpinned tabs?_"
- Actually that is not my intention. Why not pinning tabs which shall remain open. That's exactly the use case for pinning tabs.

**Comment by wolcen** [comment]
I think that this, and #42 hint at the same sort of confusion. People often will compare tab behavior to how their browser handles tabs, where pinning is obviously a rather different option. Perhaps contrasting it with typical browser behavior in README.md would help?

I may be wrong (again - pretty new to it), but it seems to be that Pinning is more equivalent to doing something in a browser like "pin tab + request a new tab for anything unopened." A new user probably would think of "pinning" to mean **ONLY** keep this in a tab "forever"/until unpinned.

In other words: since there appears to be no "open in new tab" (without explicitly "pinning" it [correct?]) I can see how that might be confusing for a new user.

**Comment by dbogdanov** [comment]
I have switched to [Joplin Favorites](https://github.com/benji300/joplin-favorites) which has a more intuitive tab behavior for my use case.

**Comment by benji300** [comment]
Hi @wolcen, hi @dbogdanov,
have you both tried the two new options introduced with v1.3.0?
- User option Unpin active tab behavior to select behavior when unpinning the active tab (#25)
- User option Add tab behavior to specify type (temporary or pinned) for new tabs (#35)
With them it should be possible avoid the "temporary" tab in almost all cases.

> Perhaps contrasting it with typical browser behavior in README.md would help?
I agree with you there. I recreated the behavior of a text editor because I found it the most intuitive. Since for me Joplin is more a text editor than a simple browser. However, I also learned that this is not true for most users. A short description of the behavior (and why it was implemented that way) really makes sense because of that.
